kudu/rollup.config.js

```diff
@@ -0,0 +1,37 @@
+import rollup from "rollup";
```

Comment:
Does this need a license at the top too?

Reply:
I'm not worried about it. This file was generated with autorest, and the main sdk repo doesn't seem to add it to their files either - for example: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/appservice/arm-appservice/rollup.config.js